app/controllers/ticket_purchases_controller.rb

```diff
@@ -30,6 +31,18 @@ def index
 
   private
 
+  def count_purchased_registration_tickets
+    purchases = params[:tickets][0]
+    purchased_registration_tickets = 0
+    @conference.tickets.for_registration.each do |registration_ticket|
+      purchased_registration_tickets += purchases[registration_ticket.id.to_s].to_i
+    end
```

Comment:
Okay so the problem is suppose there are more than 1 registration tickets in the conference and there there  id's are suppose 1 and 2 so purchase tickets would be something like this:
`{'1':'quantity', '2': 'quantity'}`
quantity can be 0 as well. So basically if there are more than 1 registration tickets then I suppose `tickets` will always be greater than 1 even if user doesn't buy any registration ticket.